src/renderer/base/renderer.cpp

```diff
@@ -355,8 +356,8 @@ DWORD Renderer::_timerToMillis(TimerRepr t) noexcept
         }
 
         // Add a bit of backoff.
-        // Sleep 150ms, 300ms, 450ms before failing out and disabling the renderer.
-        Sleep(renderBackoffBaseTimeMilliseconds * (maxRetriesForRenderEngine - tries));
+        // Sleep 100, 200, 400, 600, 800, 1600ms before failing out and disabling the renderer.
+        Sleep(renderBackoffBaseTimeMilliseconds * (1 << (maxRetriesForRenderEngine - tries - 1)));

```

Comment:
We could take this opportunity to swap the loop around to "for i from 0 to maxRetries". It'd be more readable IMO, particularly down here.

Reply:
I did. I moved the sleep earlier, as well, so that it doesn't sleep after it fails but before it presents the error message.

The backoff steps are only 100 200 400 800 now.

Should I shift them to be 200 400 800 1600?